### PR TITLE
Fix Dropout

### PR DESCRIPTION
**Pull request by ISosnovik**
@fchollet 
> At test time, the unit is always present and **the weights are multiplied by p**. 

From the original paper

### CODE REVIEW

**Comment by fchollet** [comment]
The scaling happens at training time.

**Comment by ozabluda** [comment]
explained in https://github.com/fchollet/keras/issues/3305
